### PR TITLE
Better tf image import

### PR DESCRIPTION
**Pull request by petersteneteg**
New version 
bild

### CODE REVIEW

File: include/inviwo/core/datastructures/transferfunction.h [hunk 1]
```diff
@@ -134,6 +134,9 @@ class IVW_CORE_API TransferFunction : public TFPrimitiveSet {
 
     virtual std::string getTitle() const override;
 
+
+    static std::vector<TFPrimitiveData> simplify(const std::vector<TFPrimitiveData>& points, double delta);
```
Comment (ResearchDaniel) [inline include/inviwo/core/datastructures/transferfunction.h]: Add documentation, what is delta? Percieved color difference?, intensity?, or for each rgba-component? How is opacity treated?
What would a good default value be?

File: src/core/datastructures/transferfunction.cpp [hunk 2]
```diff
@@ -217,48 +217,70 @@ void TransferFunction::load(const std::string& filename, const FileExtension& ex
...
 
+std::vector<TFPrimitiveData> TransferFunction::simplify(const std::vector<TFPrimitiveData>& points,
+                                                        double delta) {
+
+    if (points.size() < 3) return points;
+    std::vector<TFPrimitiveData> simple{points};
+
+    // Calculate the error resulting from using a linear interpolation between the prev and next
+    // point instead of including the current one
+    const auto error = [&](size_t i) {
```
Comment (ResearchDaniel) [inline src/core/datastructures/transferfunction.cpp:257]: Color and opacity kind of live in different spaces, so perhaps we should separate the threshold into one for color difference and one for opacity?

I think that percieved color difference would make more sense (e.g. convert the color to Luv before using the threshold)

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]
Nice! I miss some documentation about how the threshold in the simplify function and what it does.

Optional improvements:
- Use Luv color space instead of RGB when calculating the error. 
- Simplify could take in a distance computation lambda with the current being the default

**Review by r-englund — APPROVED** [review]

File: src/core/datastructures/transferfunction.cpp (hunk 2 above)
Comment (petersteneteg) [reply src/core/datastructures/transferfunction.cpp:257]: might be nice, but that's an other feature

**Review by ResearchDaniel — APPROVED** [review]

**Pull request merged**